bin/cli.js

```diff
@@ -14,9 +14,18 @@ if (argv._.length < 3) {
   process.exit(1);
 }
 
+// Parse from/to as either a Regex or a string
+const isRegexMatch = /.*\/([gimy]*)$/;
```

Comment:
This is a good point, thanks for pointing this out. So should we use:

`replace-in-file [-regex] from to some/file.js,some/**/glob.js`

`replace-in-file from [-isRegex] to some/file.js,some/**/glob.js`

I'd hoped I could add array support onto all the parameters, but might be a bit too complicated.